docs/common/gradle.md

```diff
@@ -67,24 +67,38 @@ Package name used for the database class.
 
 ----
 
-### `sourceFolders`
+### `srcDirs`
 
-Type: `ListProperty<String>`
+Type: `ConfigurableFileCollection`
 
-An collection of folders that the plugin will look in for your `.sq` and `.sqm` files.
-These folder paths are relative to your existing source set, so if you specify `listOf("db")`
-then the plugin will look into `src/main/db` or `src/commondMain/db`.
+A collection of folders that the plugin will look in for your `.sq` and `.sqm` files.
 
-Defaults to `listOf("sqldelight")`.
+Defaults to `emptySet()`.
```

Comment:
While this is technically true, I think we should document the "real" behavior: `Defaults to src/[prefix]main/sqldelight with prefix depending on the applied kotlin plugin eg common for multiplatform`

Reply:
Changed